certora/munged/SafeWebAuthnSignerFactory.sol

```diff
@@ -92,8 +92,7 @@ contract SafeWebAuthnSignerFactory is ISafeSignerFactory {
      * @param account The address of the account to check.
      * @return result True if the account has no code, false otherwise.
      */
-     // funtion is not really virtual, munged!
-    function _hasNoCode(address account) internal view virtual returns (bool result) {
+    function _hasNoCode(address account) internal view returns (bool result) {
```

Comment:
this file is not good, can we instead, is this the only reason for the munge ? If so please try
1. summarize _hasNoCode(address account) => hasNoCodeAlternative(account)
2. change in the harness file the _hasNoCode (internal override) to hasNoCodeAlternative external function 

maybe we can't call solidity directly from the summarization line, in that case define a cvl function that calls the solidity  function

Reply:
I see that we do have summary for hasNoCode, so maybe we can just remove this file

Reply:
what is the change to 'getSigner' ? 
so we dont need the change to ` _hasNoCode(address account) internal view virtual returns (bool result)` ?

Reply:
the change for getSigner is the extra munging in the harness.
so we don't need the change to  _hasNoCode(address account) internal view virtual returns (bool result) ?
Yep no need because of the summarization